### PR TITLE
141 convert chatbot markdown to html

### PR DESCRIPTION
**Pull request by AGI-CEO**
image
Markdown render works well for an MVP experience. Can proceed with recording a vid / gif to include in readme

### CODE REVIEW

File: client/src/index.css
Comment (gnprwx) [inline client/src/index.css:1]: Push this styling to tailwind in the className assigned to that element as discussed.

**Review by gnprwx — CHANGES_REQUESTED** [review]

File: client/src/index.css
Comment (AGI-CEO) [reply client/src/index.css:1]: Good catch. I made the edits following the consolidated styling conventions you introduced.

**Review by shannonscotta — APPROVED** [review]
truly incredible work

File: client/src/index.css
Comment (gnprwx) [reply client/src/index.css:1]: Hell yeah. Thanks for using the file for it.

Comment (gnprwx) [inline client/src/index.css:1]: Thank you for this.

**Review by gnprwx — APPROVED** [review]

File: client/src/style/style.js
Comment (gnprwx) [inline client/src/style/style.js:1]: Perfect.

**Review by gnprwx — APPROVED** [review]

**Pull request merged**